src/containers/BooksList.js

```diff
@@ -1,30 +1,41 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { removeBook } from '../actions/bookActions';
 import Book from '../components/Book';
 
 const BooksList = props => {
-  const { books } = props;
+  const { books, removeBook } = props;
+
+  const handleClick = id => {
+    removeBook(id);
+  };
+
```

Comment:
according to requirements, this should be called `handleBookRemove` 
[[see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)]

Reply:
Thanks Shey 👍🏽. We've made sure to change that function name from `handleClick` to `handleBookRemove`.